ocdskit/schema.py

```diff
@@ -234,5 +241,21 @@ def add_validation_properties(schema, unique_items=True, coordinates=False):
             if 'object' in schema['type']:
                 schema.setdefault('minProperties', 1)
 
-        for value in schema.values():
+        for key, value in schema.items():
+            if key == 'email':
+                value['format'] = 'email'
+            elif key in ['quantity', 'durationInDays', 'numberOfTenderers']:
+                value['minimum'] = 0
+            elif key in ['Organization', 'OrganizationReference']:
+                value['required'] = ['id', 'name']
+                value['properties']['name']['type'] = "string"
+                value['properties']['id']['type'] = "string"
+            elif key in ['Amendment', 'RelatedProcess']:
+                value['required'] = ['id']
+                value['properties']['id']['type'] = "string"
+            elif key == 'id':
+                if 'type' in value:
+                    if 'integer' in value['type']:
+                        value['type'].remove('integer')
```

Comment:
OCDS Kit (despite the name) has tools that are reusable by other JSON Schema standards (like BODS). We should put this logic in the standard repository. Since this leaves only email as key-based logic, we can move it as well.

Reply:
Sounds good - I've done that in https://github.com/open-contracting/standard/pull/1480 so I'll close this PR.